crates/core/src/db/datastore/locking_tx_datastore/committed_state.rs

```diff
@@ -510,9 +510,8 @@ impl CommittedState {
                     deletes.push(pv);
                 }
 
-                let table_name = &*table.get_schema().table_name;
-
                 if !deletes.is_empty() {
+                    let table_name = &table.get_schema().table_name;
```

Comment:
Is this related to the PR, or just a drive-by change? Is it actually an optimization, i.e. is `get_schema` expensive?